cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/constant/MeasureConstants.java

```diff
@@ -16,4 +16,33 @@ private MeasureConstants() {}
     // when indicated it will only evaluate for individual reportType or subject EvalType
     public static final String CQFM_CARE_GAP_DATE_OF_COMPLIANCE_EXT_URL =
             "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-care-gap-date-of-compliance-expression";
+
+    public static final String URL_CODESYSTEM_MEASURE_POPULATION =
+            "http://teminology.hl7.org/CodeSystem/measure-population";
+    // http://hl7.org/fhir/us/davinci-deqm/2023Jan/StructureDefinition-extension-populationReference.html
```

Comment:
Do you mean to leave these commented out, and if so, what are you trying to document?

Reply:
these were carried over from previous Measure.constants class, comments were left for posterity